source/overview/license-and-subscription.rst

```diff
@@ -71,9 +71,26 @@ You may incur retroactive charges for any unique users added that exceed the lic
 Renewing an Enterprise license
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ability to renew your license in the Customer Portal will be available in a future release.
+You can renew your self-managed Mattermost Enterprise license with a credit card if you have a standard Mattermost contract. When you renew your license, you can also increase the number of active users.
```

Comment:
```suggestion
From v5.32 onwards you can renew your self-managed Mattermost Enterprise license with a credit card if you have a standard Mattermost contract. When you renew your license, you can also increase the number of active users.

If you have not upgraded to v5.32 you can contact Support (support@mattermost.com) to renew your license.
```

Reply:
Propose - `For Mattermost Server version 5.32 and later` for consistency with other pages, e.g. https://docs.mattermost.com/deployment/advanced-permissions.html

![image](https://user-images.githubusercontent.com/13119842/106170845-e8790f00-615e-11eb-9e0e-64df350f537a.png)